README.md

```diff
@@ -1,70 +1,12 @@
-# Getting Started with Create React App
+# About the App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+When walking through the city and you see for instance a restaurant where you always wanted to you go, but at the time when a friend asks you 'Hey where do we wanna go?' you forget this one restaurant. Or just imagine, you see a poster of an amazing event somewhere and you directly know; I wanna go there with my best friend but also needs to be reminded - This app will help you!
```

Comment:
Maybe direct the user directly, like:
"Don't you hate these times: friends ask you out for dinner and you keep forgetting the name of this one restaurant you always wanted to go? .... Forget these struggles by using my App!

Reply:
Good one, makes it more active - thank you! 😊